Document/Document.ipynb

```diff
@@ -2127,6 +2127,20 @@
     "<center><p style=\"text-align:center;\">Fig2. Parameter Recommender Block Diagram</p></center>"
    ]
   },
+  {
```

Comment:
It's better to write something here for the pictures.
Maybe a title for them would be great.

Reply:
Which picture?

Reply:
Yes, my bad. I meant the formula.